pom.xml

```diff
@@ -87,19 +87,19 @@
         <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
         <maven-deploy-plugin.version>3.0.0-M2</maven-deploy-plugin.version>
         <maven-javadoc-plugin.version>3.3.2</maven-javadoc-plugin.version>
-        <maven-swagger-codegen-plugin.version>3.0.29</maven-swagger-codegen-plugin.version>
+        <maven-swagger-codegen-plugin.version>3.0.34</maven-swagger-codegen-plugin.version>
         <maven-swagger-codegen-handlebars.version>4.3.0</maven-swagger-codegen-handlebars.version>
 
         <db-h2.version>1.4.200</db-h2.version>
```

Comment:
h2 is also quite behind and contains CVEs, but fixes are in major version 2.1 :(